docs/source/model_doc/bert_japanese.rst

```diff
@@ -0,0 +1,75 @@
+.. 
+    Copyright 2020 The HuggingFace Team. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+    the License. You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations under the License.
+
+BertJapanese
+-----------------------------------------------------------------------------------------------------------------------
+
+Overview
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The BERT models trained on Japanese text.
+
+There are models with two different tokenization methods:
+
+- Tokenize with MeCab and WordPiece and this requires some extra dependencies, `fugashi
+  <https://github.com/polm/fugashi>`__ which is a wrapper around `MeCab <https://taku910.github.io/mecab/>`__.
+- Tokenize into characters.
+
+To use `MecabTokenizer`, you should ``pip install transformers["ja"]`` (or ``pip install -e .["ja"]`` if you install
+from source) to install dependencies.
+
+See `details on cl-tohoku repository <https://github.com/cl-tohoku/bert-japanese>`__.
+
+Example of use:
+
+.. code-block::
+
+  import torch
+  from transformers import AutoModel, AutoTokenizer 
+
+  # Example 1: Tokenize with MeCab and WordPiece
+  bertjapanese = AutoModel.from_pretrained("cl-tohoku/bert-base-japanese")
+  tokenizer = AutoTokenizer.from_pretrained("cl-tohoku/bert-base-japanese")
+  # Input Japanese Text
+  line = "吾輩は猫である。"
+
+  inputs = tokenizer(line, return_tensors="pt")
+
+  # >>> print(tokenizer.decode(inputs['input_ids'][0]))
+  # [CLS] 吾輩 は 猫 で ある 。 [SEP]
+
+  outputs = bertjapanese(**inputs)
+
+  # Example 2: Tokenize into characters
```

Comment:
Same comment as before: this could introduce the second example after the split.